torch/_inductor/ir.py

```diff
@@ -2105,6 +2106,123 @@ def realize(self):
         pass
 
 
+class BufferList(IRNode):
```

Comment:
Does this class need to exist?  We already support ops that take a list of tensor (e.g. torch.cat) and we also support ops that return a list (see `MulitOutput`).

Reply:
Removed